src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/PricingType.php

```diff
@@ -141,10 +140,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label_tag_name' => 'h2',
                 'label_help_box' => $this->trans('You can set specific prices for customers belonging to different groups, different countries, etc.', 'Admin.Catalog.Help'),
```

Comment:
```suggestion
                'label_help_box' => $this->trans('Set specific prices for customers meeting certain conditions.', 'Admin.Catalog.Help'),
```

I would advise shortening this sentence a bit. :-)